ui/app/AppLayouts/CommunitiesPortal/views/CommunitiesGridView.qml

```diff
@@ -84,9 +84,10 @@ StatusScrollView {
             popularity: model.popularity
             categories: tagsJson.model
 
-            // Community restriccions
+
+            // Community restrictions
             rigthHeaderComponent: CommunityPermissionsRow {
-                visible: !!card.permissionsList
+                visible: !!card.permissionsList && card.permissionsList.count > 0
```

Comment:
What if it will be an open community with admin permissions only? Or with channel permissions only?

Reply:
I think the component handles it. The permission model contains the type of permission and if it's hidden.

I added this additional condition, because all the communities have a `permissionsList` model, but they are empty. Without my change, a lock icon appeared on the card with no token icon